fuzz/fuzz_targets/fuzz_int_set.rs

```diff
@@ -525,124 +832,175 @@ impl Operation for IntersectOp {
     }
 }
 
-/* ### End of Ops ### */
+/* ### Invert ### */
 
-fn read_u8(data: &[u8]) -> (Option<u8>, &[u8]) {
-    if data.is_empty() {
-        return (None, data);
+struct InvertOp();
```

Comment:
totally academic but you can declare a 'unit struct' that has no fields with the syntax, `struct InvertOp;`.